contracts/src/integrations/cert/EigenDACertVerifier.sol

```diff
@@ -67,20 +68,20 @@ contract EigenDACertVerifier is
 
     constructor(
         IEigenDAThresholdRegistry initEigenDAThresholdRegistry,
-        IEigenDASignatureVerifier initEigenDASignatureVerifier,
+        IRegistryCoordinator _registryCoordinator,
         DATypesV1.SecurityThresholds memory initSecurityThresholds,
         bytes memory initQuorumNumbersRequired
-    ) {
+    ) BLSSignatureChecker(_registryCoordinator) {
         if (initSecurityThresholds.confirmationThreshold <= initSecurityThresholds.adversaryThreshold) {
             revert InvalidSecurityThresholds();
         }
         if (initQuorumNumbersRequired.length == 0 || initQuorumNumbersRequired.length > 256) {
             revert InvalidQuorumNumbersRequired(initQuorumNumbersRequired.length);
         }
         _eigenDAThresholdRegistry = initEigenDAThresholdRegistry;
-        _eigenDASignatureVerifier = initEigenDASignatureVerifier;
         _securityThresholds = initSecurityThresholds;
         _quorumNumbersRequired = initQuorumNumbersRequired;
+        staleStakesForbidden = true;
```

Comment:
can we add a comment explaining that this field lives within the inherited `BLSSignatureChecker` and how setting it to true increases security posture?